index.html

```diff
@@ -918,7 +918,7 @@ <h4>How to make tiles wrap with CSS?</h4>
     </div>
   </section>
 
-  <section id="contact" class="section has-background-light">
+  <section id="contact" class="section has-background-light" aria-label="our contacts">
```

Comment:
We usually use ‘aria-labelledby’ and point it to the section's title to tag to avoid the repetition

Reply:
Changed to aria-labelledby for all sections on the page
Thank you